schema.graphql

```diff
@@ -1046,6 +1073,20 @@ input AddDatasetSchemaInput {
   representativefield: ID
 }
 
+input CreateTagItemInput{
+  sceneId: ID!
+  label: String!
+  linkedDatasetSchemaID: ID
+  linkedDatasetID: ID
+  linkedDatasetField: ID
+}
+
+input CreateTagGroupInput{
+  sceneId: ID!
+  label: String!
```

Comment:
```suggestion
  label: String
```

When the user does not want to update the name, the user won't set the label field

Reply:
Note: apply this suggestion only for UpdateTag*Input